index.js

```diff
@@ -186,7 +186,16 @@ client.on("message", message => {
 			message.channel.send(formatLeaderboard(orderedList));
 		}
 	} else if (command == "help") {
-		message.channel.send("Commands: `!rank`, `!oldrank`, `!oldleaders`, `!help`");
+		const helpEmbed = new Discord.MessageEmbed()
+        	.setTitle("Commands")
```

Comment:
mixed in dentation here - should be tabs

Reply:
Fixed now unless Github messed with it